www/artists/jcowin/works.html

```diff
@@ -6,7 +6,390 @@
+        <link rel="stylesheet" href="css/slideshow.css">
+		 <!-- Slideshow container -->
+<div class="slideshow-container">
+
+  <!-- Full-width images with number and caption text -->
+  <div class="mySlides fade">
+    <div class="numbertext">1 / 53</div>
+    <img src="img/000.jpg" style="width:100%">
```

Comment:
You still don't have any alts on any of your images. This is not acceptable, as it causes the HTML validator to fail.